src/CmsAdmin/CategoryController.php

```diff
@@ -58,7 +62,9 @@ public function indexAction(Request $request)
     {
         $parentCategory = null;
         //wyszukiwanie parenta
-        if ($request->parentId && (null === $parentCategory = (new CmsCategoryQuery)->findPk($request->parentId))) {
+        if ($request->parentId && (null === $parentCategory = (new CmsCategoryQuery)
```

Comment:
chyba niepotrzebnie przekomplikowany ten `if` - zwłaszcza, że powyżej jest ta zmienna znullowana, a tu jest ponownie przypisana do czegoś co może być nullem - chyba wystarczyłoby to przypisanie dać przed `if`?